docs/samples/browser-plugin-meetings/app.js

```diff
@@ -409,25 +409,25 @@ function verifyPassword() {
     meeting
       .verifyPassword(joinOptions.pin, joinOptions.captcha)
       .then((res) => {
-        if (res.failureReason === 'NONE') {
+        if (res.isPasswordValid) {
```

Comment:
Instead of checking the `failureReason` checking the `isPasswordValid` and `requiredCaptcha` fields